xmtp_mls/src/storage/encrypted_store/group_message.rs

```diff
@@ -140,6 +147,13 @@ impl DbConnection {
             query = query.filter(dsl::delivery_status.eq(status));
         }
 
+        if let Some(dir) = direction {
```

Comment:
This is going to change the default behavior (if None is specified it won't be sorted at all). 

Would be a safer change if we default to ASC so we don't surprise any devs who might leave the sort direction out (or us)